i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-telephony-alcatel-oxe-snmp.md

```diff
@@ -2,48 +2,231 @@
 id: hardware-telephony-alcatel-oxe-snmp
 title: Alcatel OXE
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-## Prerequisites
 
-### Centreon Plugin
+## Contenu du Pack
+
+### Modèles
+
+Le Plugin Pack Centreon **Alcatel OXE** apporte un modèle d'hôte :
+
+* HW-Telephony-Alcatel-OXE-SNMP-custom
+
+Il apporte les modèles de service suivants :
 
-Install this plugin on each needed poller:
+| Alias             | Modèle de service                               | Description                                                                                                                                                                  | Défaut |
+|:------------------|:------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-------|
+| Cpu               | HW-Telephony-Alcatel-OXE-Cpu-SNMP               | Contrôle du taux d'utilisation du CPU de la machine. Ce contrôle pourra remonter la moyenne du taux d'utilisation des CPU ainsi que le taux par CPU pour les CPU multi-coeur | X      |
+| Disk-Generic-Id   | HW-Telephony-Alcatel-OXE-Disk-Generic-Id-SNMP   | Contrôle du taux d'espace libre disponible du disque. Pour chaque contrôle apparaîtra le nom du disque                                                                       |        |
+| Disk-Generic-Name | HW-Telephony-Alcatel-OXE-Disk-Generic-Name-SNMP | Contrôle du taux d'espace libre disponible du disque. Pour chaque contrôle apparaîtra le point de montage du disque                                                          |        |
+| Disk-Global       | HW-Telephony-Alcatel-OXE-Disk-Global-SNMP       | Contrôle du taux d'espace libre disponible des disques. Pour chaque contrôle apparaîtra le point de montage des disques                                                      |        |
+| Domain-Usage      | HW-Telephony-Alcatel-OXE-Domain-Usage-SNMP      | Contrôle le nombre d'appels externes par domaine via SNMP                                                                                                                    | X      |
+| Memory            | HW-Telephony-Alcatel-OXE-Memory-SNMP            | Contrôle du taux d'utilisation de la mémoire vive                                                                                                                            | X      |
+| PBX-Role          | HW-Telephony-Alcatel-OXE-PBX-Role-SNMP          | Contrôle du taux d'utilisation de la mémoire vive                                                                                                                            | X      |
```

Comment:
wrong service description?

Reply:
Oww, will correct the Pack accordingly; Thanks!